wavefront/server/modules/plugins_module/plugins_module/controllers/cloud_storage_controller.py

```
@@ -0,0 +1,102 @@
+    except Exception as e:
+        return JSONResponse(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            content=response_formatter.buildErrorResponse(str(e)),
+        )
+
+
+@cloud_storage_router.get('/v1/storage/read')
+@inject
+async def read_storage_file(
+    resource_url: str = Query(..., description='The cloud storage URL of the resource'),
+    type: StorageFileType = Query(StorageFileType.json, description='File type'),
```

Ruff (0.15.2)

[warning] Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)